### DISCUSSION TITLE
is there any way to remove dropped items

### DISCUSSION THREAD

**Discussion by PercyGFX** [Nimdy/Dedicated_Valheim_Server_Script#58] [Q&A]
im using a small server. so i think there will be a performance issue while these unwanted items laying around.

is there any way to remove dropped items using a command ?

**Comment by devdavi** [comment]
Not directly related to this script, but if you've set yourself up as an admin you can hit `f5` to access the console and use the command `removedrops`  to delete all drops in the area (not the entire server). 

I have not tested this in game, but that's what I've seen on it.

**Reply by PercyGFX** [reply]
thank you
